scripts/update_online.py

```diff
@@ -0,0 +1,27 @@
+import socket
+import os
+def arg(p=1):
+  try:
+    import sys
+    return sys.argv[p]
+  except:
+    return None
+os.chdir("..")
+
+lists = ["porn.txt","abuse.txt","ads.txt","basic.txt","crypto.txt","drugs.txt","everything.txt","facebook.txt","fraud.txt","gambling.txt","malware.txt","phishing.txt","piracy.txt","ransomware.txt","redirect.txt","scam.txt","smart-tv.txt","tiktok.txt","torrent.txt","tracking.txt","whatsapp.txt","youtube.txt"]
+for list in lists:
+    file = open(list)
+    alt = open(list.split(".")[0] + "_lite.txt","w")
+    lines = file.read().split("\n")
+    for line in lines:
+        if line.startswith("#") or "127.0.0.1" not in line:
```

Comment:
Not understanding the `or "127.0.0.1" not in line` part.

Reply:
This is to detect empty lines (lines without an entry)

Reply:
Can you give a concrete example of where you see empty lines in the lists?

Reply:
I don't think there is an empty line, but I can switch it to 
```py
if line.startswith("#") or line == '':
```
to avoid an error if there ever was an empty line, as one empty line would probably crash the entire program

Reply:
Apparently I had to leave the `127.0.0.1 not in` part or else it gets into an infinite loop